proto/iotics/api/meta.proto

```diff
@@ -108,6 +114,28 @@ message SparqlQueryResponse {
   // Headers for the query result. clientRef within can be used to identify which query the result applies to.
   Headers headers = 1;
 
-  // SPARQL result chunk payload.
+  // SPARQL query result chunk payload.
+  Payload payload = 2;
+}
+
+// Performs a SPARQL update against custom metadata only.
+message SparqlUpdateRequest {
+
+  // SPARQL update request headers
+  Headers headers = 1;
+
+  // SPARQL update request payload.
+  message Payload {
+    // A UTF8-encoded SPARQL 1.1 query
+    bytes update = 1;
```

Comment:
`updateChunk`?

We probably should indicate somehow that:
- hosts will have a maximum accepted message size configured (e.g. fully wrapped is [4MiB by default](https://pkg.go.dev/google.golang.org/grpc#MaxRecvMsgSize) - which in reality would thus mean something strictly lower than that)
- the client should split larger update queries accordingly and send them in multiple messages
- the query will only be deemed complete once the send-direction of the stream is closed (e.g. `CloseSend` [in Golang]( https://pkg.go.dev/google.golang.org/grpc#ClientStream))

Reply:
by way of comments you mean?

Reply:
Yes. And I guess beyond that there'll be documentation.